dataminer/Administrator_guide/DataMiner_Agents/Configuring_a_DMA/Configuring_the_IP_network_ports.md

```diff
@@ -43,15 +43,23 @@ A DataMiner System makes extensive use of TCP/IP communication. Below, you find
 
 ## Graphical representation of IP communication within a DMS
 
-The diagrams below show how communication within a DMS could be set up. The blue lines indicate the communication towards the databases (in this case an OpenSearch and Cassandra cluster of three nodes each).
+### Recommended setup
 
-Using gRPC (recommended from DataMiner 10.3.6/10.3.0 [CU3] onwards):
+The image below shows how communication within a DataMiner System should be set up, when you use DataMiner with STorage as a Service (STaaS). The DMAs in the cluster communicate via gRPC (recommended from DataMiner 10.3.6/10.3.0 [CU3] onwards) over HTTPS port 443.
 
-![DMS communication overview new](~/dataminer/images/dms_ip_communication_with_DB_updated.png)
+![DMS communication - STaaS](~/dataminer/images/Connection_Overview1.svg)
 
-Using .NET Remoting:
+If a server running a DxM does not have CloudGateway installed, any outgoing communication to dataminer.services will go through a CloudGateway available elsewhere in the cluster. In the example below, this means server 2 uses the CloudGateway on server 1 over port 5100.
 
-![DMS communication overview](~/dataminer/images/dms_ip_communication_with_DB.png)
+![Multiple servers - CloudGateway](~/dataminer/images/CloudGatewayServers.svg)
+
+*\*IP list: A list of IP addresses that must be allowed through the firewall. For STaaS, this refers to the [Microsoft Azure IP ranges](https://www.microsoft.com/en-us/download/details.aspx?id=56519). For dataminer.services, see [Connecting to dataminer.services](xref:Cloud_connectivity_and_security#connecting-to-dataminerservices).*
```

Comment:
@LouisSkyline The list of Microsoft Azure IP ranges (`ServiceTags_Public_20251124.json`) contains some of the IP addresses used for STaaS (`*.windows.net`) but not the IP addresses used for `cloudstorage-*.dataminer.services`. The [STaaS docs page](https://docs.dataminer.services/dataminer/Troubleshooting/Procedures/STaaS_Troubleshooting.html#verify-your-dma-has-a-working-internet-connection) mentions that you should receive an email, are those specified inthere?